docs/user/QUICKSTART.md

```diff
@@ -16,39 +16,43 @@ Install the Gateway API CRDs:
 kubectl apply -f https://github.com/envoyproxy/gateway/releases/download/v0.2.0-rc2/gatewayapi-crds.yaml
 ```
 
-Run Envoy Gateway:
+Deploy Envoy Gateway:
 
 ```shell
 kubectl apply -f https://github.com/envoyproxy/gateway/releases/download/v0.2.0-rc2/install.yaml
 ```
 
-Run the example app:
+Deploy the example app:
 
 ```shell
 kubectl apply -f https://raw.githubusercontent.com/envoyproxy/gateway/v0.2.0-rc2/examples/kubernetes/httpbin.yaml
 ```
 
-The Gateway API resources must be created in the following order. First, create the GatewayClass:
+The Gateway API resources must be created in the following order:
 
-```shell
-kubectl apply -f https://raw.githubusercontent.com/envoyproxy/gateway/v0.2.0-rc2/examples/kubernetes/gatewayclass.yaml
-```
+1. First, create a GatewayClass:
```

Comment:
The quickstart related yaml was squashed into a single yaml, so we'll just need to apply that to create gwclass, gw, httproute, service, deployment.